### PR TITLE
Incrementing the version number to 0.14.0

### PR DESCRIPTION
**Pull request by antalszava**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+### Improvements
+
+* Updated the test suite for environments with IBMQX tokens and for the new
+  core of PennyLane.
+  [(#14)](https://github.com/PennyLaneAI/pennylane-orquestra/pull/14/files)
+  [(#17)](https://github.com/PennyLaneAI/pennylane-orquestra/pull/17/files)
+
+This release contains contributions from (in alphabetical order):
+
+Josh Izaac, Antal Száva.
```
Comment (josh146) [inline CHANGELOG.md:12]:
```suggestion
Antal Száva.
```
I would say my contribution was negligible here :)

File: steps/test_expval.py [hunk 2]
```diff
@@ -176,10 +176,15 @@ def circuit():
...
-        ops, _ = dev.process_observables(qnode.circuit.observables)
+        with monkeypatch.context() as m:
+            m.setattr(dev, "execute", lambda *args, **kwargs: np.array([None]))
+            qnode = qml.QNode(circuit, dev)
+            qnode()
+
+        circuit = qnode.qtape.graph if hasattr(qnode, "qtape") else qnode.circuit
+
+        qasm_circuit = dev.serialize_circuit(circuit)
+        ops, _ = dev.process_observables(circuit.observables)
```
Comment (josh146) [inline steps/test_expval.py:187]: How come this wasn't failing previously?

**Review by josh146 — APPROVED** [review]
Thanks Antal!

File: steps/test_expval.py (hunk 2 above)
Comment (antalszava) [reply steps/test_expval.py:187]: This is testing the server-side part and was pulling PennyLane `0.13.0` from a different `requirements.txt` file than the one that was pinned to master 😕

**Pull request merged**